articles/gitops-mode.md

```diff
@@ -0,0 +1,38 @@
+# GitOps mode
+
+The Fleet UI now supports GitOps mode, which helps users avoid unexpected changes when using GitOps
+to configure Fleet by preventing users from manually updating
+GitOps-configurable settings and features in the UI.
+
+For example, if a user uses the UI to add
+a new saved query and then runs GitOps, the manual change will be overwritten by the GitOps
+configuration. To help avoid this potentially confusing situation, GitOps mode prevents the user
+from manually saving or editing the query in the first place (though does still allow running an ad-hoc live query):
```

Comment:
```suggestion
For example, if a user in the Fleet UI adds a query and then GitOps runs, the query will be deleted. GitOps helps avoid this by presenting the user from saving or editing the query in the first place (though does still allow running an ad-hoc live query):
```

What do you think? Also trying to cut down on words here.

Reply:
Much better